### PR TITLE
Added ccloud CLI setup instructions

### PR DESCRIPTION
**Pull request by theduderog**
Add instructions for setting up `ccloud` CLI

### CODE REVIEW

File: ccloud/README.md [hunk 1]
```diff
@@ -12,11 +12,186 @@ of the Confluent Platform documentation.
...
+### Windows
+
+Download and unzip from [here](https://s3-us-west-2.amazonaws.com/confluent.cloud/cli/ccloud-latest.zip).  Use the bin/ccloud.ps1 Powershell script to run ccloud CLI.
+
+## Setup
+
+To get started you will need to configure ccloud by running the `init` command.
+
+```shell
+$ ./bin/ccloud init
```
Comment (travisjeffery) [inline ccloud/README.md]: it should be in your path i think. should just be $ ccloud init

File: ccloud/README.md [hunk 2]
```diff
@@ -12,11 +12,186 @@ of the Confluent Platform documentation.
...
+### Mac OS X
+
+```shell
+$ brew tap confluentinc/ccloud
+$ brew install ccloud
+```
+
+### Linux
+
+Download and unzip from [here](https://s3-us-west-2.amazonaws.com/confluent.cloud/cli/ccloud-latest.tar.gz).  Add the bin directory to your PATH.
```
Comment (travisjeffery) [inline ccloud/README.md]: should this say add the contents of your bin directory to your path?

Comment (travisjeffery) [reply ccloud/README.md]: rather than than the directory itself

**Comment by lokesh-shekar** [comment]
@theduderog Does this override the one in KnowledgeBase?. We should have one place maybe.

File: ccloud/README.md [hunk 3]
```diff
@@ -12,11 +12,186 @@ of the Confluent Platform documentation.
...
+	Topic: testCustom	Partition: 13	Leader: 14	Replicas: 14,3,4	Isr: 14,3,4
+	Topic: testCustom	Partition: 14	Leader: 15	Replicas: 15,4,5	Isr: 15,4,5
+	Topic: testCustom	Partition: 15	Leader: 16	Replicas: 16,5,6	Isr: 16,5,6
+	Topic: testCustom	Partition: 16	Leader: 17	Replicas: 17,6,7	Isr: 17,6,7
+	Topic: testCustom	Partition: 17	Leader: 0	Replicas: 0,7,8	Isr: 0,7,8
+	Topic: testCustom	Partition: 18	Leader: 1	Replicas: 1,9,10	Isr: 1,9,10
+	Topic: testCustom	Partition: 19	Leader: 2	Replicas: 2,10,11	Isr: 2,10,11
+```
+
+Now let's produce some items into the `test` topic.
```
Comment (chicagobuss) [inline ccloud/README.md]: nit:  items => messages

File: ccloud/README.md (hunk 2 above)
Comment (chicagobuss) [reply ccloud/README.md]: +1

File: ccloud/README.md (hunk 3 above)
Comment (theduderog) [reply ccloud/README.md]: thx

**Pull request merged**